tests/data/use_braces_in_int_method_calls.py

```diff
@@ -0,0 +1,24 @@
+count = 5 .bit_count()
```

Comment:
What about complex numbers? They don't require parens (`10j.imag` is valid), so it would be good to test that we don't add it.

Similarly `0x10.imag` is valid, so let's test that we don't add unnecessary parens.

Reply:
It even works on floats, but on floats, the multiple "dot operators" look a bit confusing, so I decided to keep it on floats
```py
>>> 10.1.is_integer()
False
```

Right will do, `10j.imag` are `0x10.imag` are fine with the parentheses 👍🏻